lib/public/views/Tags/ActiveColumns/index.js

```diff
@@ -11,37 +11,48 @@
  * or submit itself to any jurisdiction.
  */
 
+import { formatTimestamp } from '../../../utilities/formatTimestamp.js';
+
 /**
  * Method to retrieve the list of active columns for a generic table component
  * @param {Object} model The global model object
  * @return {Object} A collection of columns with parameters for the Tag table
  */
 const activeColumns = () => ({
     id: {
-        name: 'Entry ID',
-        visible: true,
-        size: 'w-10',
+        visible: false,
         primary: true,
-        sortable: false,
     },
     text: {
         name: 'Name',
         visible: true,
-        size: 'w-40',
+        classes: 'w-40 f6',
         sortable: false,
     },
+    lastEditedName: {
+        name: 'Last Edited by',
+        visible: true,
+        classes: 'w-10 f6',
+        format: (name) => name || '-',
+    },
+    updatedAt: {
+        name: 'Updated at',
+        visible: true,
+        size: 'w-10 f6',
+        classes: formatTimestamp,
+    },
     mattermost: {
         name: 'Mattermost',
         visible: true,
-        size: 'w-20',
-        format: (mattermost) => mattermost ? mattermost : '-',
-        sortable: false,
+        size: 'w-20 f6',
```

Comment:
I mixed up things, but my point is I do not know why we call it size, because OK it define a size but it is only a class name, and in the table code it is simply used as class name so I proposed to rename it. But, this will again add a divergence, so it is up to you.
If we would like to separate the width from the classes, I do not think it is a good idea to put w- prefix in front of it, because it couple it to the css behind the scene